src/ReinforcementLearningCore/src/policies/learners/abstract_learner.jl

```diff
@@ -9,7 +9,7 @@ Base.show(io::IO, m::MIME"text/plain", L::AbstractLearner) = show(io, m, convert
 
 # Take Learner and Environment, get state, send to RLCore.forward(Learner, State)
 function forward(L::Le, env::E) where {Le <: AbstractLearner, E <: AbstractEnv}
-    env |> state |> Flux.gpu |> (x -> forward(L, x)) |> Flux.cpu
+    env |> state |> (x -> forward(L, x))
```

Comment:
This is minor, and not even yours, but the Julian style is to use lower case for variables and start with a capital for types. I think L::Le should become "learner".